Assets/Scripts/Behaviors/IAttacker.cs

```diff
@@ -0,0 +1,4 @@
+﻿public interface IAttacker
+{
```

Comment:
Keep the pull request only for the feature that is requested in the assignment

Reply:
Removed.